packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/information/information.html

```diff
@@ -0,0 +1,92 @@
+<oui-modal
+    data-heading="{{:: 'pci_databases_pools_information_title' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+>
+    <div class="mb-3 information">
+        <dl>
+            <div class="row my-2 py-2">
+                <dt
+                    class="col-sm-3 font-weight-bold"
+                    data-translate="pci_databases_pools_information_database"
+                ></dt>
+                <dd
+                    class="col-sm-9 my-auto"
+                    data-ng-bind="$ctrl.pool.database.name"
+                ></dd>
+            </div>
+            <div class="row my-2 py-2 border-top">
+                <dt
+                    class="col-sm-3 font-weight-bold"
+                    data-translate="pci_databases_pools_information_port"
+                ></dt>
+                <dd
+                    class="col-sm-9 my-auto"
+                    data-ng-bind="$ctrl.database.port"
+                ></dd>
+            </div>
+            <div class="row my-2 py-2 border-top">
+                <dt
+                    class="col-sm-3 font-weight-bold"
+                    data-translate="pci_databases_pools_information_ssl_mode"
+                ></dt>
+                <dd
+                    class="col-sm-9 my-auto text-capitalize"
+                    data-ng-bind="$ctrl.database.sslMode"
+                ></dd>
+            </div>
+            <div class="row my-2 py-2 border-top">
+                <dt class="col-sm-12">
```

Comment:
**issue(blocking):** `.row > .col-*-12` doesn't make sense. Use a regular `div` (if needed).
Could you please take it into account for this template as you have applied multiple time following structure `.row > .col-*-12`.